homeassistant/components/simplisafe/__init__.py

```diff
@@ -454,17 +448,21 @@ def _async_process_new_notifications(self, system):
 
     async def async_init(self):
         """Initialize the data class."""
-        asyncio.create_task(self.websocket.async_connect())
+        # 2021-04-29: Disabling connection to the websocket due to the SimpliSafe cloud
```

Comment:
Instead of commenting it out, remove the full functionality

Reply:
Why? That's a huge code change (beyond what this PR currently covers) and there's a non-0 possibility that I could bring it back if I figure out what SimpliSafe is doing.

If the concern is that this comment will stay in the codebase forever, I can commit to a time by which I come back and clean up if a new solution isn't found.

Reply:
Ok then. Fix the lint issue and let's put it in the beta.

Reply:
Appreciate it, @balloob. I'll commit to removing everything by 2021.6.0 if I haven't figured out a way forward.